test/plugin/scenarios/servicecomb-2.x-scenario/support-version.list

```diff
@@ -0,0 +1,19 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+2.6.0
+2.6.1
+2.6.2
```

Comment:
No, and only major version should be tested in upstream. Such as 2.6.2(if it is the latest) should be kept, and all others should be removed. You could test locally, but not in the stream codebase because all tests of plugins have to run in every pull request. We just keep meanful ones.